### PR TITLE
Feature/bocm 486 create file storage service

### PR DESCRIPTION
**Pull request by mayya-bond-kc**
## Describe your changes

- Minor updates to where the params for the endpoints are sent
- Additional documentation
- Dockerfile addition

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/document-storage/src/server/app/routes.js [hunk 1]
```diff
@@ -1,21 +1,80 @@
...
-import { validateDocumentUpload, validateGetAllDocuments, validateUploadDocument, validateDocumentName } from './validator.js';
 import { downloadDocument, getAllDocuments, uploadDocument } from './controller.js';
+import { asyncHandler } from './middleware/async-handler.js';
+import { validateDocumentName,validateGetAllDocuments, validateUploadDocument } from './validator.js';
 
-const router = express.Router();
+const router = new express.Router();
 
 router.get('/', 
+	/*
```
Comment (francescobrenci-AR) [inline apps/document-storage/src/server/app/routes.js:9]: is this commented out code still necessary? - if so can you add a comment please

File: apps/document-storage/src/server/swagger-output.json [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                "type": "string",
+                "example": "application"
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
```
Comment (francescobrenci-AR) [inline apps/document-storage/src/server/swagger-output.json:139]: new line at the end of the file needed

**Review by DavidJWhiteDev — DISMISSED** [review]

File: apps/document-storage/src/server/app/routes.js (hunk 1 above)
Comment (mayya-bond-kc) [reply apps/document-storage/src/server/app/routes.js:9]: this is what generates the api documentation in swagger. so we need this in there

**Review by francescobrenci-AR — DISMISSED** [review]

**Review by francescobrenci-AR — APPROVED** [review]

**Pull request merged**